warrior/WarriorCore/Classes/war_cli_class.py

```diff
@@ -320,7 +320,11 @@ def parser(cls, arglist):
         parser.add_argument('-datafile', action='store', nargs='?',
                             help="overwrite the path of datafile in execution "\
                             "ignore the datafile specified in testcase.xml")
-
+        #to accept -testwrapperfile as command line argument
```

Comment:
replace testwrapperfile with wrapper file.  When specified from command line it will skip both suite.xml and also testcase.xml. Update the help string.

Reply:
done